apps/meteor/tests/end-to-end/api/09-rooms.js

```diff
@@ -1142,7 +1142,7 @@ describe('[Rooms]', function () {
 			});
 		});
 		it('should return an error when the required parameter "selector" is not provided', (done) => {
-			updatePermission('can-audit', ['admin']).then(() => {
+			updatePermission('can-audit', ['admin']).then(
```

Comment:
Are the changes here because of prettier or? 👀

Reply:
Yeah, likely because of prettier. Not needed so removing them